tests/integration/ssa.sh

```diff
@@ -360,9 +364,17 @@ echo "========================================================================"
 seeds="47 147 1147"
 methods="0 1 2"
 frag_sz=0
+file_format="graphml"
+
+if [ ${file_format} == "graphml" ] ; then
+  file_ext="graphml"
+  prob_simple="${WCS_TEST_DIR}/problem/Gillespie/eq29-exprtk.${file_ext}"
+elif [ ${file_format} == "sbml" ] ; then
+  file_ext="xml"
+  prob_simple="${WCS_TEST_DIR}/problem/Gillespie/eqn29-sbml.${file_ext}"
+fi
 
-prob_simple="${WCS_TEST_DIR}/problem/Gillespie/eq29-exprtk.graphml"
-prob_complex="${WCS_TEST_DIR}/problem/Birtwhistle/birtwistle.rate.graphml"
+prob_complex="${WCS_TEST_DIR}/problem/Birtwhistle/birtwistle.rate.${file_ext}"
```

Comment:
If prob_complex does not exist for sbml, we can print warning, or error.
Especially, in ssa_performance test, it should print error because the test requires prob_complex

Reply:
There is already a check  at line 394 if any of these file does not exist.

Reply:
Should be good then